From: Maximiliano Sandoval <msandova@gnome.org>
Date: Tue, 18 Mar 2025 18:06:18 +0100
Subject: widget: Explicitly annotate visible's getter

Since
https://gitlab.gnome.org/GNOME/gobject-introspection/-/merge_requests/473
g-i will pick up gtk_widget_is_visible as the getter for this property.

See https://gitlab.gnome.org/GNOME/gjs/-/issues/681.

(cherry picked from commit 9b3476aaf672616d28dbb525e5a6ce39d9a533de)

Origin: upstream gtk-3-24 after 3.24.49
---
 gtk/gtkwidget.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/gtk/gtkwidget.c b/gtk/gtkwidget.c
index a565e58..b6d125e 100644
--- a/gtk/gtkwidget.c
+++ b/gtk/gtkwidget.c
@@ -1148,7 +1148,9 @@ gtk_widget_class_init (GtkWidgetClass *klass)
                         -1, G_MAXINT,
                         -1,
                         GTK_PARAM_READWRITE|G_PARAM_EXPLICIT_NOTIFY);
-
+  /**
+   * GtkWidget:visible: (getter get_visible)
+   */
   widget_props[PROP_VISIBLE] =
       g_param_spec_boolean ("visible",
                             P_("Visible"),
